### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by angrycuban13**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+title: ''
```
Comment (nemchik) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: What does title do?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+labels: ''
```
Comment (nemchik) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: This can probable use the `bug` label.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (nemchik) [inline .github/ISSUE_TEMPLATE/feature-request.md:5]: This can probably use the `enhancement` label

**Review by nemchik — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (angrycuban13) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: I have no idea lol. I think if we wanted a specific title like `[BUG] Blah blah blah`

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 3 above)
Comment (angrycuban13) [reply .github/ISSUE_TEMPLATE/feature-request.md]: 1299787

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (angrycuban13) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 1299787

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (angrycuban13) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 1299787

**Review by nemchik — APPROVED** [review]

**Pull request merged**